projects/netgrif-components/src/lib/admin/ldap-group-role-assignment/ldap-group-role-assignment.component.html

```diff
@@ -0,0 +1,137 @@
+<div fxLayout="row" fxLayoutAlign="start center" class="full-height full-width padding-default">
```

Comment:
how customizable is this component? We have already run into issues with the role assignment component, where replacing some small part of it necessitates copying the entire implementation into the developed application in order to make the changes.

Would we prefer a more modular approach in combination with schematics, or are we happy with this implementation for the moment?

Reply:
I think both components need to be redesigned